src/main/resources/templates/uploadDocuments.html

```diff
@@ -159,12 +112,11 @@ <h3 class="h4" th:text="#{upload-documents.proof-of-income}"></h3>
             myDropZone = this;
             this.on('addedfile', file => {
                 const fileNameSpan = file.previewElement.getElementsByClassName('filename-text')[0]
-                // const fileNameComponents = getFilenameComponents(fileNameSpan.innerText);
-                fileNameSpan.innerText = truncateFileName(fileNameSpan);
-                //
-                // const formattedHtml = "<span class='filename-text-name'>"+fileNameComponents[0] + "</span>" +
-                //     "<span class='filename-text-ext'>"+fileNameComponents[1] + "</span>"
-                // fileNameSpan.innerHTML = formattedHtml
+                fileNameSpan.setAttribute('aria-label', fileNameSpan.innerText)
+                const fileNameComponents = getFilenameComponents(fileNameSpan.innerText);
+                const formattedHtml = "<span class='filename-text-name' aria-hidden='true'>"+fileNameComponents[0] + "</span>" +
+                    "<span class='filename-text-ext' aria-hidden='true'>."+fileNameComponents[1] + "</span>"
```

Comment:
This may not be needed, but it looks like elements have a "scrollWidth" and a "clientWidth", and if the scrollWidth is longer, then we know there is an overflow so we wouldnt need the "." before the file extension in that case

Reply:
Good find!